packages/example/dapp/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ### Patch Changes
 
+-   @wallet-standard/react@0.1.2
+
+## null
```

Comment:
Hmm, something weird happening here.

Reply:
It [has always been `null` for this package](https://github.com/wallet-standard/wallet-standard/blob/bca63b5a964135fc847fa75925884c312e706195/packages/example/react/CHANGELOG.md) (and any other package in the `example` directory) as the `version` defined in `package.json` for them is `null`. For this reason, we should probably not block on this.

Additionally, could we consider removing the `CHANGELOG` files from the examples?